File: DDS.WebApi/Models/Review.cs
#nullable disable

using System;

namespace DDS.WebApi.Models
{
    public partial class Review
    {
        public int ReviewId { get; set; }
        public int OwnershipId { get; set; }
        public int Rating { get; set; }
        public DateTime Date { get; set; }
        public string Comment { get; set; }

        public virtual Ownership Ownership { get; set; }
    }
}

File: DDS.Client/ClientApp/src/app/models/review.ts
import { Ownership } from "./ownership";
import { User } from "./user";

export class Review {

  constructor(
    public reviewId?: number,
    public ownershipId?: number,
    public rating?: number,
    public date?: Date,
    public comment?: string,
    public ownership?: Ownership
  ) {
  }
}
